src/esp/sensor/PinholeCamera.cpp

```diff
@@ -28,8 +29,37 @@ void PinholeCamera::setProjectionParameters(SensorSpec::ptr spec) {
   hfov_ = std::atof(spec_->parameters.at("hfov").c_str());
 }
 
-void PinholeCamera::setProjectionMatrix(gfx::RenderCamera& targetCamera) {
+PinholeCamera& PinholeCamera::setProjectionMatrix(
+    gfx::RenderCamera& targetCamera) {
   targetCamera.setProjectionMatrix(width_, height_, near_, far_, hfov_);
+  return *this;
+}
+
+PinholeCamera& PinholeCamera::setModelViewMatrix(
+    gfx::RenderCamera& targetCamera) {
+  Magnum::Matrix4 T = this->node().absoluteTransformation();
+  Magnum::Matrix3 R = T.rotationScaling();
+  Magnum::Math::Algorithms::gramSchmidtOrthonormalizeInPlace(R);
+
+  VLOG(1) << "||R - GS(R)|| = "
+          << Eigen::Map<mat3f>((R - T.rotationShear()).data()).norm();
+
+  T = Magnum::Matrix4::from(R, T.translation()) *
```

Comment:
I just copied and pasted whatever we had to the new functions.
Good suggestion. I will change them.

Reply:
Yeah... I wrote this code and I go back and forth on uppercase variables names for matrices.  Uppercase is pretty much the universal way to specify a matrix in math/linear algebra, so that is why I do it.